### ISSUE TITLE
my pokebuddy doesnt login anymore???

### ISSUE THREAD

**Issue by hashkikker** [finndev/PokeBuddy#657]
i start it up type in FREE-BETA set character name and pass put walk speed to 12km/h and start but nothing happens..

help me please i already re downloaded it

**Comment by Sukize** [comment]
Have the same Problem

**Comment by curtiola** [comment]
No, he means pokebuddy doesn't log in, you mean your ptc account doesn't log in, I don't know how to solve the free-standing log in not working sadly

**Comment by hashkikker** [comment]
yeah it doesnt log in  while using auth provider
